README.md

```diff
@@ -831,6 +831,334 @@ to the `assets/css/app.css` file:
 }
 ```
 
+## 5. Adding authentication 🔐
```

Comment:
@LuchoTurtle please start adding whitespace between a section heading and its' content
the way _all_ our repos do, e.g: https://github.com/dwyl/learn-tdd/blame/main/README.md#L17

Yes, it's technically "valid" Markdown to _not_ have whitespace between the header and the content ...
but it doesn't _look_ right. 

Think of it like this, a "Mankini" is _technically_ a valid form of swimwear ... 🩱 
![image](https://user-images.githubusercontent.com/194400/203656771-ecf75312-0679-4ef7-9243-9abf7adaa65b.png)

But it doesn't _look_ right, does it? 😬 

```suggestion
## 5. Adding authentication 🔐

```

Reply:
You're correct, I should follow the convention used across dwyl's repos. 
But personally, to me, it looks "right" without whitespace.

It's just a matter of preference :)